src/open_producten/producttypen/models/producttype.py

```diff
@@ -0,0 +1,113 @@
+from datetime import date
+
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from markdownx.models import MarkdownxField
+
+# from open_producten.locations.models import Contact, Location, Organisation
+from open_producten.utils.models import BasePublishableModel
+
+from .onderwerp import Onderwerp
+from .upn import UniformeProductNaam
+
+
+class OnderwerpProductType(models.Model):
+    """
+    Through-model for Onderwerp-ProductType m2m-relations.
+    """
+
+    onderwerp = models.ForeignKey(Onderwerp, on_delete=models.CASCADE)
+    product_type = models.ForeignKey("ProductType", on_delete=models.CASCADE)
+
+
+class ProductType(BasePublishableModel):
+    naam = models.CharField(
+        verbose_name=_("naam"),
+        max_length=100,
+        help_text=_("naam van het product type."),
+    )
+
+    samenvatting = models.TextField(
+        verbose_name=_("samenvatting"),
+        default="",
+        max_length=300,
+        help_text=_("Korte beschrijving van het product type, maximaal 300 karakters."),
+    )
+
+    beschrijving = MarkdownxField(
+        verbose_name=_("beschrijving"),
+        help_text=_("Product type beschrijving met WYSIWYG editor."),
+    )
+
+    keywords = ArrayField(
+        models.CharField(max_length=100, blank=True),
+        verbose_name=_("Keywords"),
+        default=list,
+        blank=True,
+        help_text=_("Lijst van keywords waarop kan worden gezocht."),
+    )
+
+    uniforme_product_naam = models.ForeignKey(
```

Comment:
Can there or would there be multiple product types per UPN?

I guess something like regular passport and express passport since they give the same product but different prices?

Reply:
Yes there can be multiple, although prices are stored on the same product.